### PR TITLE
Trying to make documentation less Docker Centric

### PR DESCRIPTION
**Pull request by rhatdan**
We should talk about
Container Registries
Container Images
and Containers, rather then prefixing everything with Docker.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by kalexand-rh** [comment]
@vikram-redhat, thoughts?

**Comment by vikram-redhat** [comment]
@rhatdan @kalexand-rh this requires sign-off from @mpbarrett and @aheslin at the very least.

File: admin_guide/diagnostics_tool.adoc [hunk 1]
```diff
@@ -90,7 +90,7 @@ Available diagnostics include:
 |Check systemd service logs for problems. Does not require a configuration file to check against.
 
 |`ClusterRegistry`
-|Check that the cluster has a working Docker registry for builds and image streams.
+|Check that the cluster has a working container registry for builds and image streams.
```
Comment (smarterclayton) [inline admin_guide/diagnostics_tool.adoc]: We say "image registry", not "container registry" in product explanations.

**Comment by smarterclayton** [comment]
This is the general direction.  I'll add a few comments though about explicit naming.

File: rest_api/oapi/v1.Image.adoc [hunk 2]
```diff
@@ -11,7 +11,7 @@ toc::[]
 
 == Description
 [%hardbreaks]
-Image is an immutable representation of a Docker image and metadata at a point in time.
+Image is an immutable representation of a container image and metadata at a point in time.
```
Comment (smarterclayton) [inline rest_api/oapi/v1.Image.adoc]: These are automatically generated and can't be changed like this (rest_api)

File: release_notes/ose_3_2_release_notes.adoc [hunk 3]
```diff
@@ -133,7 +133,7 @@ issue.
 header the new `*userAgentMatching*` configuration setting.
 
 [[ose-integrated-docker-registry]]
-==== Integrated Docker Registry
+==== Integrated Container Registries
```
Comment (smarterclayton) [inline release_notes/ose_3_2_release_notes.adoc]: `Integrated Container Image Registry` singular

**Comment by kalexand-rh** [comment]
I took @smarterclayton's suggestions, removed the changes to the rest_api topics, and made a new PR here: https://github.com/openshift/openshift-docs/pull/12357

**Comment by kalexand-rh** [comment]
Per @vikram-redhat, I'm closing this PR and will merge #12357 and apply it to 3.11.

**Pull request closed without merging**